src/PointwiseFunctions/AnalyticData/GrMhd/CylindricalBlastWave.cpp

```diff
@@ -18,10 +19,11 @@
 
 namespace {
 template <typename DataType>
-Scalar<DataType> compute_piecewise(
-    const tnsr::I<DataType, 3, Frame::Inertial>& x, const double& inner_radius,
-    const double& outer_radius, const double& inner_value,
-    const double& outer_value) noexcept {
+Scalar<DataType> compute_piecewise(const tnsr::I<DataType, 3>& x,
+                                   const double& inner_radius,
+                                   const double& outer_radius,
+                                   const double& inner_value,
+                                   const double& outer_value) noexcept {
```

Comment:
I don't believe there is an advantage to taking these doubles by reference, and AFAIK doing so adds the expense of a pointer dereference... I suggest taking them by value instead.